addons/account/models/account_journal_dashboard.py

```diff
@@ -486,19 +486,18 @@ def _fill_general_dashboard_data(self, dashboard_data):
         if not general_journals:
             return
         to_check_vals = {
-            vals['journal_id'][0]: vals
-            for vals in self.env['account.move'].read_group(
+            journal: (count, amount_total_signed_sum)
+            for journal, count, amount_total_signed_sum in self.env['account.move']._read_group(
                 domain=[('journal_id', 'in', general_journals.ids), ('to_check', '=', True)],
-                fields=['amount_total_signed'],
-                groupby='journal_id',
-                lazy=False,
+                groupby=['journal_id'],
+                aggregates=['__count', 'amount_total_signed:sum'],
```

Comment:
```suggestion
                groupby=['journal_id'],
                aggregates=['amount_total_signed:sum', '__count'],
```
Same, or swap to_checks